### DISCUSSION TITLE
real time or post process?

### DISCUSSION THREAD

**Discussion by elitan** [deepgram/community#988] [General help] [answered]
Are there any quality differences between getting the "final results" in real-time VS doing a full audio file transcription (not real-time)?

I'm asking because I'm trying to decide whether to use real-time final results or transcribe a conversation after it's completed. I'm looking for the solution with the highest accuracy.

**Comment by deepgram-community** [comment]
Hey there! It looks like you haven't connected your GitHub account to your Deepgram account. You can do this at https://community.deepgram.com - being verified through this process will allow our team to help you in a much more streamlined fashion.

**Comment by deepgram-community** [comment]
It looks like we're missing some important information to help debug your issue. Would you mind providing us with the following details in a reply?

- The programming language you are working in (e.g. JavaScript, Python).
- The deepgram product you are using (e.g Speech to Text, Agent API)
- A request ID that triggered your error or issue.

**Comment by jkroll-deepgram** [comment, marked as answer]
Hi @elitan, Deepgram's pre-recorded transcription is slightly more accurate that streaming, because there is more context available across the audio. For reference, we have [benchmarked](https://deepgram.com/learn/nova-2-speech-to-text-api) our English pre-recorded transcription at 8.4% word error rate (WER), compared to streaming transcription at 10.7% WER. So if you don't require real-time transcription, using pre-recorded transcription will give you modest accuracy gains.
